server/vulnerabilities/cpe.go

```diff
@@ -159,10 +159,10 @@ func CPEFromSoftware(db *sqlx.DB, software *fleet.Software) (string, error) {
 	case "portage_packages":
 	case "rpm_packages":
 	case "npm_packages":
-		targetSW = "node.js"
+		targetSW = `"node.js"`
 	case "atom_packages":
 	case "programs":
-		targetSW = "windows*"
+		targetSW = `"windows*"`
```

Comment:
```go
targetSW = `"windows"*` // asterisk outside quotes
```

Reply:
Hm, but it does match windows_8 and so on. Is it a nit? or am I missing something?

Reply:
Ok, my mistake there, looks like we are good:
```
sqlite> SELECT count(*) FROM cpe WHERE rowid in (SELECT rowid FROM cpe_search WHERE target_sw MATCH '"windows"*') order by deprecated asc;
4250

sqlite> SELECT count(*) FROM cpe WHERE rowid in (SELECT rowid FROM cpe_search WHERE target_sw MATCH '"windows*"') order by deprecated asc;
4250
```